pkg/infra/infra.go

```diff
@@ -145,8 +145,13 @@ var _ = Describe("[Feature:Machines] Managed cluster should", func() {
 	var machineSet *machinev1.MachineSet
 	var machineSetParams framework.MachineSetParams
 
+	gatherer, err := framework.NewGatherer()
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
You might have issues with calling an expectation outside of a `BeforeEach`/`It`, you probably want to `var gatherer framework.Gatherer` and then run this bit of the code in the `BeforeEach`

Reply:
Yeah, agree. Done.